listenbrainz/webserver/static/js/src/user/Listens.tsx

```diff
@@ -687,19 +688,30 @@ export default class Listens extends React.Component<
               />
             )}
             <ListenCountCard user={user} listenCount={listenCount} />
-            {user && (
-              <div
-                className="card hidden-xs hidden-sm"
-                style={{ paddingTop: "1.5em" }}
-              >
-                <UserSocialNetwork user={user} newAlert={newAlert} />
-              </div>
-            )}
+            {user && <UserSocialNetwork user={user} newAlert={newAlert} />}
           </div>
           <div className="col-md-8 col-md-pull-4">
             {!listens.length && (
-              <div className="lead text-center">
-                <p>No listens yet</p>
+              <div className="empty-listens">
+                <FontAwesomeIcon icon={faRecordVinyl as IconProp} size="10x" />
+                <div className="lead empty-text">Empty record</div>
+                <div className="empty-message text-muted">
+                  {currentUser?.name === user?.name
+                    ? "You haven't"
+                    : `${user.name} hasn't`}{" "}
+                  listened to any songs yet.
+                </div>
+                {currentUser?.name === user?.name && (
+                  <div className="empty-action">
+                    Import <a href="/profile/import/">your listening history</a>{" "}
```

Comment:
Maybe spelling this out? What do you think?
```suggestion
                    Import <a href="/profile/import/">your listening history</a> from Last.fm/Libre.fm{" "}
```